src/scripts/components/notification.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Single Notification UI component
+ * https://github.com/WordPress/wp-feature-notifications/issues/16#issuecomment-896031592
+ * https://github.com/WordPress/wp-feature-notifications/issues/37#issuecomment-896080025
+ */
+import { __ } from '@wordpress/i18n';
+
+// Register components
+import { Component } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import NoticeImage from './noticeImage';
+
+import moment from 'moment';
+import { delay } from '../utils';
+
+// Notice class method.
+export default class Notice extends Component {
```

Comment:
Does this have to be a class? Or could it just be `const` or `function`? It's not got any class-specific logic or anything.

Reply:
For certain things I have always preferred to create a class whereas I use functional components more often for component elements, this one has also a method (onDismiss) and I guess more may be added in the future..
In this way the component is also more prone to being extended (not that it would be impossible otherwise)